docs/subgraph/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Aragon Subgraph [WIP]
+sidebar_label: Introduction
+sidebar_position: 0
+---
+
+## The On-Chain Information
+
+In Aragon we use [The Graph](https://thegraph.com/docs/en/about/), a decentralized protocol for indexing and querying blockchain data. The Graph makes it possible to query data that is difficult to retrieve directly from the Smart Contracts.
```

Comment:
```suggestion
Subgraph is a protocol for indexing and querying event data emitted from the smart contracts.
```
https://www.alchemy.com/overviews/what-is-a-subgraph

The Graph is a decentralized subgraph API. We use Alchemy subgraphs which is a more centralized version that is cheaper.
Both index events that are stored in the transaction logs on the blockchain. Please suggest more improvements @Rekard0.